### PR TITLE
Operators Chat Group FE v0

### PR DESCRIPTION
**Pull request by abjjabjj**
I've added the FE to our codebase in `webapp/main`. If you all have any ideas for a better name feel free to suggest.

In any case, the application can be spun up locally via
```make local-webapp-main```

For local development, this shares the same sqlite database as the concrete API, so I would recommend spinning that up to manipulate the database via
```make local-api```

You'll need the refresh the FE to see any changes. Currently there's only dynamic loading for orchestrators since messages are hardcoded and the API for it doesn't exist yet.

image
image

### CODE REVIEW

**Comment by kgang** [comment]
1. Read things from sqllite
2. Create objects inside the frontend

**Comment by kgang** [comment]
`/chat` kind of works? All I see is the Executive saying 'I am the Executive!'
`/orchestrators` does not work

**Comment by kgang** [comment]
I see orchestrators in my db at the api's orchestrators endpoint

**Comment by abjjabjj** [comment]
`/chat` is still hard-coded

**Comment by abjjabjj** [comment]
do you see the list of orchestrators at root?

**Comment by kgang** [comment]
I see nothing at root

**Comment by kgang** [comment]
the favicon loads tho :star-struck:

**Comment by abjjabjj** [comment]
lol nice

File: webapp/main/server.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+app.mount("/static", StaticFiles(directory=os.path.join(dname, "static")), name="static")
+
+
+@app.get("/", response_class=HTMLResponse)
+async def root(request: Request):
+    return templates.TemplateResponse("index.html", {"request": request})
+
+
+@app.get("/chat", response_class=HTMLResponse)
+async def chat(request: Request):
```
Comment (kgang) [inline webapp/main/server.py:25]:
```suggestion
async def chat(request: Request):
    # TODO: Make dynamic
```

**Review by kgang — APPROVED** [review]
LG2M

**Comment by MichaelDeng03** [comment]
image
Orchestrators looks a little messed up.
This is the corresponding orchestrator in my db. 
image
Root looks good though.

dym writing up some additional notes on how to whip up a new orchestrator/other sqlmodel objects via the api? I ended up using orm.crud directly in my terminal, which was a huge pain. 

I'm also not 100% sure what file path sql_app.db ends up at. It seems like your FE uses sql_app.db in the root directory, but I often do db operations in my scratch notebook directory, and a sql_app.db instance also gets instantiated in there.

File: webapp/main/server.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    # TODO: make dynamic, both in data and in template.
+    messages = [
```
Comment (MichaelDeng03) [inline webapp/main/server.py:27]: Should remove messages if it's not being used for templating

**Pull request merged**